### ISSUE TITLE
send: incremental send fails on / 

### ISSUE THREAD

**Issue by mirco** [kdave/btrfs-progs#122]
Pull Request #98 doesn't handle the case of mnt being / properly, causing incremental sends from the rootfs to fail with
ERROR: not on mount point: /
even if both the subvolume to send and the parent subvolume given with -p are located on /.

Steps to reproduce:
```
btrfs subvolume snapshot -r / /parent
btrfs subvolume snapshot -r / /child
btrfs send -p /parent /child > /dev/null
```

**Comment by stralytic** [comment]
I'm also having this problem with btrfs-progs 4.16.1 on ubuntu cosmic

**Comment by kdave** [comment]
Fixed by #138 and will be released in 4.17.
